packages/common/src/hooks/useListData/useSortBy.ts

```diff
@@ -0,0 +1,39 @@
+import { useCallback, useState } from 'react';
```

Comment:
seems i'm messing this diff up. This has a change described above, where I thought it would be good to just pass a key. Will need to change this back to pass {key, isDesc}